CONTRIBUTING.md

```diff
@@ -27,8 +27,8 @@ your new repository and run the build scripts:
 
 You need to have some supported version of Python installed and have
 it available as `python` in your shell. To run Zookeeper you also
-need a Java runtime (JRE or JDK) version 6 or 7. To run tests, you
-need to have the tox, Python testing tool, to be installed in your shell.
+need a Java runtime (JRE or JDK) version 6, 7 or 8. To run tests, you
```

Comment:
Is this dependency on JDK/JRE 6 or 7 hardcoded anywhere? I added 8 now that it's in GA, but personally I'd rather remove the version numbers altogether as it's one less thing that needs updating.